### ISSUE TITLE
About the 'strict' parameter when generating the pq codes

### ISSUE THREAD

**Issue by hyc9** [RUCAIBox/VQ-Rec#6]
Thanks for your inspiring work!  When I check the code for generating PQ codes, I noticed that there was a 'strict' option, it seems to be used for deciding whether to train the index using the data from the training set, or using all the item features to train the index space. I am not entirely sure if my understanding is correct, and I would appreciate it if you could provide further explanation on why this distinction is made and the effects of these two different approaches.
I don't seem to have found a detailed explanation of this parameter, so if possible please let me know if there are any errors or omissions in my understanding, really appreciate that ！

**Comment by hyp1231** [comment]
Hi! Thanks for your attention to our work.

Yes, you are right. If the parameter `strict` is `True`, we will only use item text from the training set to train PQ codes. That's to simulate a real-world scenario and prevent potential data leakage. In the real world, we can only obtain the item text from the training set, and those new items from the valid set and test set are invisible.

**Comment by hyc9** [comment]
I see, thanks !
